src/vg_chain_state.erl

```diff
@@ -9,7 +9,8 @@
 
 -export([start_link/0,
          role/0,
```

Comment:
nothing actually seems to use the role anymore now that it's getting passed around everywhere.  remove maybe?